### DISCUSSION TITLE
TIC-80 Raspberry Pi Pro Version

### DISCUSSION THREAD

**Discussion by SRHMan** [nesbox/TIC-80#1729] [General]
I have purchased the pro version of TIC-80 and am kind of disappointed that there isn't an easy to install .deb file of the pro version for the raspberry pi in the downloads like there is for the demo version.  I've tried building it myself  by using "cmake .. -DBUILD_PRO=On" and I'm getting some build errors that I'm not really in the mood to wade through.  I've downloaded all the dependencies and follow instructions from various sites.  I would think that the raspberry pi is popular enough that you could include a Debian package file for it.

**Comment by nesbox** [comment]
Hmm, it's strange, these are the same .deb packages, only the binary file is different. Maybe, try to uninstall the demo version first, or you can unpack the pro .deb and just replace the `tic80` binary.

**Comment by SRHMan** [comment]
I'm sorry, but where do I get the Raspberry Pi .deb package for the PRO version?  It doesn't seem to on itch.io download page for the pro version where I purchased it.  Just the Android, windows, Linux (I assume for X64), and Mac OS.  Would it be that hard to make one to put with the other versions?

**Comment by nesbox** [comment]
It's not, will add pro with the upcoming release.
THanks

**Comment by SRHMan** [comment]
Cool.  Thanks so much.

**Comment by nvictor** [comment]
Hello, I am trying to get a small single board computer to run TIC-80 on. Is it confirmed that it runs fine on the Raspberry Pi now?

Thanks.
